src/todos/todos.service.ts

```
@@ -191,35 +226,62 @@ export class TodosService {
       console.log(error);
-      throw new UnauthorizedException("Invalid TODO Update")
+
+      return {
+        result: 'error',
+        message: 'INVALID_REQUEST'
+      }
     }
   }
 
-  async remove(todo_id: number, user_id: number) {
+  async remove(todo_id: number, user_id: number): Promise<SuccessResponse | FailResponse | ErrorResponse> {
```

ESLint

[error] Replace `todo_id:·number,·user_id:·number` with `⏎····todo_id:·number,⏎····user_id:·number,⏎··`

(prettier/prettier)